### ISSUE TITLE
Funding ideas

### ISSUE THREAD

**Issue by evrifaessa** [FOSSBilling/FOSSBilling#11]
- Donations and sponsorships through Open Collective
- **Maybe** advertisements in our main website (there seems to be great alternatives to Google Adsense. Such as [EthicalAds by Read The Docs](https://www.ethicalads.io/advertising-vision/). The money would go to our Open Collective account, and we'd make sure to go with something that won't track our users, or don't place advertisements at all.

FOSSBilling is a free project, and will likely be free as long as it exists. We'll need a valid economical sustainibility plan. Let's discus this here, transparently.

**Comment by BelleNottelling** [comment]
Potentially further down the line, paid / priority support could provide some further income.

The big thing imo is keeping the entire project open source and free. If someone doesn't want to pay for it, I still want them to have full functionality

**Comment by evrifaessa** [comment]
> The big thing imo is keeping the entire project open source and free. If someone doesn't want to pay for it, I still want them to have full functionality

Absolutely. That's why we removed Pro from BoxBilling in the first place and I personally wouldn't want to have that back in the future.

**Comment by jaapmarcus** [comment]
> * **Maybe** advertisements in our main website (there seems to be great alternatives to Google Adsense. Such as [EthicalAds by Read The Docs](https://www.ethicalads.io/advertising-vision/). The money would go to our Open Collective account, and we'd make sure to go with something that won't track our users, or don't place advertisements at all.

For the low traffic not worth the time to setup...

**Comment by gOOvER** [comment]
Paid Support for Buisness  Customers ;) THey earn Money and when they need help, they should pay :)

**Comment by notbasetwo** [comment]
I think that sponsorships is definitely the way to go, and certainly the most realistic! For people who would like a specific feature - having it be prioritised by someone throwing some money at it is always a bonus. 

Paid support would probably be an option once the product is in a much cleaner shape than it is now - I think at the moment it would be too much to ask for given the state of the project.

Advertising on the site would likely not be worth a huge ton, perhaps if there was a good chunk of informative content (i.e. tutorials, etc) - that could drive traffic & ad revenue.
